src/test/java/it/gov/pagopa/arc/service/bizevents/BizEventsServiceImplTest.java

```diff
@@ -206,5 +229,78 @@ void givenTransactionIdWhenCallRetrieveTransactionReceiptFromBizEventsThenReturn
         Mockito.verify(bizEventsConnectorMock).getTransactionReceipt(anyString(),anyString());
     }
 
+    @Test
+    void givenParametersWhenCalRetrievePaidListFromBizEventsThenReturnPaidList() {
+        //given
+        BizEventsPaidNoticeDTO bizEventsPaidNoticeDTO = BizEventsPaidNoticeDTOFaker.mockInstance(1, false);
+        List<BizEventsPaidNoticeDTO> listOfbizEventsPaidNoticeDTO = List.of(bizEventsPaidNoticeDTO);
+
+        BizEventsPaidResponseDTO bizEventsPaidResponseDTO = BizEventsPaidResponseDTO.builder().notices(listOfbizEventsPaidNoticeDTO).continuationToken(CONTINUATION_TOKEN).build();
+
+        NoticeDTO noticeDTO = NoticeDTOFaker.mockInstance(1, false);
+        List<NoticeDTO> listOfNoticeDTO = List.of(noticeDTO);
+
+        NoticesListDTO noticesListDTO = NoticesListDTO.builder().notices(listOfNoticeDTO).build();
+
+        NoticesListResponseDTO noticesListResponseDTO = NoticesListResponseDTO.builder().noticesListDTO(noticesListDTO).continuationToken(CONTINUATION_TOKEN).build();
+
+        when(bizEventsPaidNoticeConnectorMock.getPaidNoticeList(DUMMY_FISCAL_CODE,CONTINUATION_TOKEN,SIZE, true, true, ORDER_BY, ORDERING)).thenReturn(bizEventsPaidResponseDTO);
+        when(bizEventsPaidNoticeDTO2NoticeDTOMapperMock.toNoticeDTOList(listOfbizEventsPaidNoticeDTO)).thenReturn(listOfNoticeDTO);
+        when(bizEventsPaidNoticeListDTO2NoticesListDTOMapperMock.toNoticesListDTO(listOfNoticeDTO)).thenReturn(noticesListDTO);
+        when(bizEventsPaidResponseDTO2NoticesListResponseDTOMapperMock.toNoticesListResponseDTO(noticesListDTO, CONTINUATION_TOKEN)).thenReturn(noticesListResponseDTO);
+        //when
+        NoticesListResponseDTO result = bizEventsService.retrievePaidListFromBizEvents(CONTINUATION_TOKEN, SIZE, true, true, ORDER_BY, ORDERING);
+
+        //then
+        Assertions.assertNotNull(result);
+        assertEquals(1, result.getNoticesListDTO().getNotices().size());
+        assertEquals(noticesListResponseDTO.getNoticesListDTO().getNotices().get(0), result.getNoticesListDTO().getNotices().get(0));
+        assertEquals(CONTINUATION_TOKEN, result.getContinuationToken());
+
+
+        Mockito.verify(bizEventsPaidNoticeConnectorMock).getPaidNoticeList(anyString(),anyString(),anyInt(),anyBoolean(),anyBoolean(),anyString(),anyString());
+        Mockito.verify(bizEventsPaidNoticeDTO2NoticeDTOMapperMock).toNoticeDTOList(any());
+        Mockito.verify(bizEventsPaidNoticeListDTO2NoticesListDTOMapperMock).toNoticesListDTO(anyList());
+        Mockito.verify(bizEventsPaidResponseDTO2NoticesListResponseDTOMapperMock).toNoticesListResponseDTO(any(), anyString());
```

Comment:
`verify` is not necessary if configuring the `when` condition.
Moreoever the `when` condition are also specific! These verify are not giving nothing more to the test, we could remove them